src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/ResourceManagementOptions.cs

```diff
@@ -1,22 +1,20 @@
-using System;
-
 namespace OrchardCore.ResourceManagement
 {
     public class ResourceManagementOptions
     {
-        public bool UseCdn { get; set; } = true;
+        public bool UseCdn { get; set; }
```

Comment:
I suggest to leave the default values here, and use these default in the `SiteSettings`

Reply:
@hishamco this has been done, as appropriate, without extra project referencing.

Reply:
How?!! coz may I use the `ResourceManagement` outside Orchard Core

Reply:
How?!! coz may I use the `ResourceManagement` outside Orchard Core